pennylane/transforms/optimization/cancel_inverses.py

```diff
@@ -64,6 +63,45 @@ def _are_inverses(op1, op2):
     return False
 
 
+def _can_cancel_ops(op1, op2):
+    """Checks if two operators can be cancelled
+
+    Args:
+        op1 (~.Operator)
+        op2 (~.Operator)
+
+    Returns:
+        Bool
+    """
+    if _are_inverses(op1, op2):
```

Comment:
Can we also update this to `_non_wires_inverses`?

Reply:
Changed locally